src/lib/list/list.ts

```diff
@@ -50,6 +53,26 @@ export class MdList {
   set disableRipple(value: boolean) { this._disableRipple = coerceBooleanProperty(value); }
 }
 
+@Component({
+  moduleId: module.id,
+  selector: 'md-selection-list, mat-selection-list',
+  host: {'role': 'listbox'},
+  template: '<ng-content></ng-content>',
+  styleUrls: ['list.css'],
+  encapsulation: ViewEncapsulation.None
+})
+export class MdSelectionList {
```

Comment:
In another PR after this one, we'll need to add support for the `MdSelectionList` being a form control, similar to the `md-select`